provisioner/roles/manage_ec2_instances/tasks/create_inventory.yml

```diff
@@ -33,3 +33,7 @@
 - name: f5 mode - setup inventory
   include_tasks: "addhost_f5.yml"
   when: workshop_type == 'f5'
+
+- name: Security Automation - setup inventory
```

Comment:
Please rename to `security mode - setup inventory` to follow the naming guideline of all other inventories.